### DISCUSSION TITLE
Vite项目中L7Layer无法导入

### DISCUSSION THREAD

**Discussion by lisentowind** [antvis/L7#2535] [Q&A]
Vite项目中从@antv/l7-leaflet无法导入L7Layer,报错504  更换Vite4或者5 都是报错，从@antv/l7中导入 LineLayer没有问题

![code-ts](https://github.com/antvis/L7/assets/88653280/49c16bcd-f951-4f34-8adf-6ead525a2a7a)

#### 部分依赖版本
1. "@antv/l7": "^2.22.0"
2.  "@antv/l7-leaflet": "^1.0.2"
3.  "leaflet": "^1.9.4"

**Comment by Zqiqi-q** [comment]
遇到了同样的问题，希望得到解决！
